scripts/ci/build-installers.sh

```diff
@@ -48,8 +48,8 @@ if [ "$ARGV_OPERATING_SYSTEM" == "linux" ]; then
   ./scripts/build/docker/run-command.sh \
     -r "$TARGET_ARCH" \
     -s "$(pwd)" \
-    -c 'make installers-all'
+    -c 'make electron-develop installers-all'
```

Comment:
> So wouldn't it be better to just "fix" the optional dependencies sanity check in some way?

Trust me I tried, but couldn't find a way to do so without some non-trivial effort. I think we can avoid the whole thing by making a program that parses the shrinkwrap files instead of simply taking a list of optional modules to operate, but that'll take time, and I want to get this transition done ASAP so we can focus on other stuff.

I'll try to get something up and running while these PRs get merged. If I finish fast enough, I'll send a PR and we can rebase this one. What do you think?

> (when you say "not needed" above, did you actually mean "not installed"?)

So let's say there's an optional dependency for GNU/Linux called `foo`. When you run `electron-develop`, `foo` gets installed, however when attempting to check that the shrinkwrap is up to date, `foo` will get removed in OSes other that GNU/Linux. If the tasks that follow need to make use of this module, they will fail with ENOENT errors on GNU/Linux.

> BTW, if this really is a problem, how come we never noticed it before???

Looks like none of the optional dependencies that were getting removed were needed to generate the installers.